### PR TITLE
Updates README.md

### PR DESCRIPTION
**Pull request by jvanz**
## Description
Updates the README.md content adding the list of the repositories with their scope and status. The README.md file also has links to other useful documentation.

Related to https://github.com/kubewarden/kubewarden-controller/issues/659

**NOTE**: some links in this files requires other PR to be merge. However, we can merge it as it is. The link will work when all the PR got merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,192 @@
...
+ - [Repositories](#repositories)
+   * [Repositories Guidelines](./REPOSITORIES.md)
+   * [Core](#core)
+   * [Infra](#infra)
+   * [Policies](#policies)
+   * [Special](#special)
+   * [Archived](#archived)
+ - [Contributing](#contributing)
+ - [Security policy](#security-policy)
+ - [Join the Community](#join-the-community)
```
Comment (robertsirc) [inline README.md:24]: where does this go?

**Review by robertsirc — APPROVED** [review]
This reads just fine. The only question is the link to join the community is that a CTA and where does that go?

File: README.md (hunk 1 above)
Comment (jvanz) [reply README.md:24]: To the other sections in the README file.

Comment (jvanz) [reply README.md:24]: I just noticed that there are some broken links. I'll fix it.

Comment (jvanz) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -1 +1,189 @@
...
+
+## Code Of Conduct
+
+We follow the [CNCF Code of
+Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+Please contact
+[cncf-kubewarden-maintainers@lists.cncf.io](mailto:cncf-kubewarden-maintainers@lists.cncf.io)
+or the Linux Foundation mediator, Mishi Choudhary
+[mishi@linux.com](mailto:mishi@linux.com) to report an issue.
```
Comment (olblak) [inline README.md:33]: I am surprised that we need to specify a specific person address.
What if that person stop working for the LF?

**Review by olblak — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (olblak) [reply README.md:33]: Just to clarify that my comment is not a blocker but just a notice. Feel free to close it if we can't use a "generic" email address

Comment (jvanz) [reply README.md:33]: Good question, I was following the Falco example and just copy this paragraph. But you're right. I need to double check if I should do that.

Comment (jvanz) [reply README.md]: Reading the CNCF documentation I found better information to use in this section. Therefore, I've updated the content. ;)

File: README.md [hunk 3]
```diff
@@ -1 +1,193 @@
...
+| [kubewarden/policy-sdk-go](https://github.com/kubewarden/policy-sdk-go)                                                                            | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              Kubewarden Policy SDK for the Go programming language|
+| [kubewarden/policy-sdk-swift](https://github.com/kubewarden/policy-sdk-swift)                                                                   | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              Kubewarden Policy SDK for the Swift programming language|
+| [kubewarden/cel-policy](https://github.com/kubewarden/cel-policy)                                                                                                  | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              A policy that can run CEL expressions|
```
Comment (viccuad) [inline README.md:142]: I think the cel-policy repo should be in incubating status for now.

**Review by viccuad — APPROVED** [review]
LGTM, just status of cel-policy and kyverno policy.

File: README.md [hunk 4]
```diff
@@ -1 +1,193 @@
...
+| [kubewarden/policy-fetcher](https://github.com/kubewarden/policy-fetcher/)              | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |             Crate used by Kubewarden that is able to pull policies from OCI registries and HTTP servers. |
+| [kubewarden/kwctl](https://github.com/kubewarden/kwctl/)              | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |          Go-to CLI tool for Kubewarden users   |
+| [kubewarden/helm-charts](https://github.com/kubewarden/helm-charts)                     | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |            Helm charts for the Kubewarden project   |
```
Comment (flavio) [inline README.md:77]: I've just put the most important repos (from a user POV) first, I've also done some small change to the policy-evaluator and policy-fetcher descriptions


```suggestion
| [kubewarden/kubewarden-controller](https://github.com/kubewarden/kubewarden-controller) | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |          Manage admission policies in your Kubernetes cluster with ease   |
| [kubewarden/policy-server](https://github.com/kubewarden/policy-server)                 | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |           Webhook server that evaluates WebAssembly policies to validate Kubernetes requests   |
| [kubewarden/audit-scanner](https://github.com/kubewarden/audit-scanner)                 | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |             Reports evaluation of existing Kubernetes resources with your already deployed Kubewarden policies|
| [kubewarden/kwctl](https://github.com/kubewarden/kwctl/)              | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |          Go-to CLI tool for Kubewarden users   |
| [kubewarden/helm-charts](https://github.com/kubewarden/helm-charts)                     | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |            Helm charts for the Kubewarden project   |
| [kubewarden/policy-evaluator](https://github.com/kubewarden/policy-evaluator)           | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |           Rust library used by Kubewarden to evaluate policies with a given input, request to evaluate and settings.   |
| [kubewarden/policy-fetcher](https://github.com/kubewarden/policy-fetcher/)              | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable) |             Rust library used by Kubewarden to pull policies from OCI registries and HTTP servers. |
```

File: README.md (hunk 3 above)
Comment (flavio) [reply README.md:142]: Yes, I agree

File: README.md [hunk 5]
```diff
@@ -1 +1,193 @@
...
+| [kubewarden/rancher-kubectl-builder](https://github.com/kubewarden/rancher-kubectl-builder)                                                           | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              Workflow to rebuild and sign rancher/kubectl image|
+| [kubewarden/github-actions](https://github.com/kubewarden/github-actions)                                                                                  | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              GitHub actions used by the Kubewarden project|
+| [kubewarden/kubewarden-end-to-end-tests](https://github.com/kubewarden/kubewarden-end-to-end-tests)                                                        | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              Files used to run Kubewarden end-to-end tests|
+
+### Policies
```
Comment (flavio) [inline README.md:90]: I would do the following changes if possible:

* Put the repos in alphabetical order (if it's not too much, I hope you used a script to generate the table, maybe we could add a `status` label to the repos and build everything via script)
* Create a new section with the policy templates:
  * Go, Rust, OPA, Gatekeeper: stable
  * DotNet: incubation
  * Swift: sandbox

File: README.md [hunk 6]
```diff
@@ -1 +1,193 @@
...
+| [kubewarden/k8s-objects](https://github.com/kubewarden/k8s-objects)                                                                       | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              Experimental: Kubernetes Go types that can be used with TinyGo|
+| [kubewarden/utils](https://github.com/kubewarden/utils)                                                                                             | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              Utils scripts used by the Kubewarden team and users.|
+| [kubewarden/policy-hub](https://github.com/kubewarden/policy-hub)                                                                                            | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              A place where to find and discover policies|
```
Comment (flavio) [inline README.md:173]: I've archived that

File: README.md [hunk 7]
```diff
@@ -1 +1,193 @@
...
+| [kubewarden/gtmpl-rust](https://github.com/kubewarden/gtmpl-rust)                                                                                                          | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |              golang text/template for rust|
+| [kubewarden/hugo-gh-pages](https://github.com/kubewarden/hugo-gh-pages)                                                                           | [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)      |             Build and Publish Your Hugo Site to GitHub Pages|
```
Comment (flavio) [inline README.md:175]: This is archived, we might even drop it

**Review by flavio — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (jvanz) [reply README.md:142]: Yes, as I said during a daily, we should review the status. I've added the same status for all the policies to make my life easier during the table writing. :)

Thanks for doing that!

**Comment by jvanz** [comment]
@kubewarden/kubewarden-documentation @kubewarden/kubewarden-developers I've added a new file to contain the security policy of the project.

**Review by jhkrug — APPROVED** [review]
lgtm

**Review by flavio — CHANGES_REQUESTED** [review]
Everything looks great. I've one final request: can you move the policy SDK repos to another dedicated section?

The page should have these section:
* ready to use policies
* policy SDKs
* policy templates

**Review by flavio — APPROVED** [review]
That's perfect :clap: :heart:

**Pull request merged**